docs/reference/runtime/Execution.md

```diff
@@ -1,6 +1,25 @@
 ## Execution
 
-You can use the same RTOS API in ISR. The only two warnings are:
+### Threads
+
+Your application (`main` function) will start execution in the main thread, but it's not the only thread running in Mbed OS. There's a number of live system threads at any given time. They should be transparent from the user's point of view for most use cases. All threads that exist in the system by default:
+* main - default thread that executes application's `main` function.
```

Comment:
main thread - ... int main void is the main system thread and has stack size of ... by default

Reply:
I've added that as a note after, but it fits here better, I'll move that info inline.